### ISSUE TITLE
[Debian] Broken CHOWN_REPLACEMENT when built on Windows

### ISSUE THREAD

**Issue by wobu** [sbt/sbt-native-packager#600]
Preconditions:
- use JDebPackaging cause anything else isn't possible on windows
- have **src/universal** files, like **src/universal/bin/test.sh**
- have **linuxPackageMappings** with different user than **root**, like:

``` scala
linuxPackageMappings in Debian := {
    (linuxPackageMappings in Debian).value.map(_ withUser "test" withGroup "test")
 }
```

-> build package

Result:
In the "postinst" file of the exploded  debian package you can see chown lines similiar to this:

``` bash
chown test:test /usr/share/pkg_name/bin\test.sh
```

Note the backslash previous to the file name.
I think the cause is that **mappings in Universal** contains the file paths with Windows File Path Separators.

Finally if you try to install the debian package you will get following error message:

``` nash
chown: cannot access ‘/usr/share/pkg_name/bintest.sh’: No such file or directory
```

where the backslash gets escaped or removed

Possible Workaround for me currently:

``` scala
linuxPackageMappings in Debian := {
      (linuxPackageMappings in Debian).value
.map(lpm => lpm.copy(mappings = lpm.mappings.map(f => f._1 -> f._2.replace("\\", "/"))))
}
```

**Comment by muuki88** [comment]
Thanks for the detailed report. 

> I think the cause is that mappings in Universal contains the file paths with Windows File Path Separators.

This is most likely the cause of this issue. Steps for fixing this would be

1. A regression test for this (we have automated windows tests)
2. Thinking of a good solution for this

### Solution

The fast way would be to integrate the string-replace (more or less) in the `DebianPlugin`.
Another solution would be to test if JDeb needs these post-inst `chmods`. I'm not sure how
JDeb supports users and permissions, but using their API would be a lot better than hacky
string-replaces.

**Comment by wobu** [comment]
seems to be solved in meantime, tested with 1.2.0
